python/llm/src/ipex_llm/transformers/npu_pipeline_model/minicpm.py

```diff
@@ -258,9 +259,20 @@ def convert_minicpm_layer(model, layer_idx, n_splits_linear, n_splits_down_proj,
                                                         f"decoder_layer_{layer_idx}",
                                                         temp_dir)
 
-    for idx, (weight, scale) in enumerate(weights):
-        bin_file = os.path.join(weight_dir, f"model_{layer_idx}_input_{5+idx*2}.bin")
-        weight.numpy().tofile(bin_file)
-        bin_file = os.path.join(weight_dir, f"model_{layer_idx}_input_{5+idx*2+1}.bin")
-        scale.numpy().tofile(bin_file)
+    if layernorm_const:
+        for idx, (weight, scale) in enumerate(weights):
```

Comment:
Maybe add a start_idx and merge the duplicated parts?

Reply:
> Maybe add a start_idx and merge the duplicated parts or make it a utility function?

Have added `st_idx` and remove duplicated part.